test/test_module.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+"""Check the attributes of edm4hep package"""
+
+import edm4hep
+from pathlib import Path
+from ROOT import gInterpreter
+
+if gInterpreter.LoadFile("edm4hep/EDM4hepVersion.h") != 0:
+    raise ImportError("Error while loading file edm4hep/EDM4hepVersion.h")
+
+# __version__
+version = edm4hep.version.build_version
+assert edm4hep.__version__ == f"{version.major}.{version.minor}.{version.patch}"
+# __name__
+name = "edm4hep"
+assert edm4hep.__name__ == name
+# __file__
+expected_origin = str(Path(__file__).parent.parent / "python" / "edm4hep" / "__init__.py")
+assert edm4hep.__file__ == expected_origin
+# __path__
+expected_path = [str(Path(expected_origin).parent)]
+assert edm4hep.__path__ == expected_path
```

Comment:
These will obviously be different after installation, but I don't see an easy way to check them without doing an install first. Anyhow, there is no reason to expect this to not work after installation, unless I am missing something?

Reply:
I didn't think about it. I think it should be possible to find package without importing it and then check if the attributes' values are the same on the imported module. That should work both in build environment and after install